hw/top_earlgrey/BUILD

```diff
@@ -91,6 +91,24 @@ fpga_cw310(
     rsa_key = {"//sw/device/silicon_creator/rom/keys/fake/rsa:test_private_key_0": "test_key_0"},
 )
 
+fpga_cw310(
+    name = "fpga_cw310_rom_ext",
+    testonly = True,
+    base = ":fpga_cw310_test_rom",
+    bitstream = "//hw/bitstream:rom_with_fake_keys",
+    exec_env = "fpga_cw310_rom_ext",
+    linker_script = "//sw/device/lib/testing/test_framework:ottf_ld_silicon_owner_slot_a",
+    manifest = "//sw/device/silicon_owner:manifest_standard",
+    param = {
+        "interface": "cw310",
+        "exit_success": DEFAULT_TEST_SUCCESS_MSG,
+        "exit_failure": DEFAULT_TEST_FAILURE_MSG,
+        "assemble": "{rom_ext}@0 {firmware}@0x10000",
```

Comment:
Any test cases requiring `rom_ext_b` will simply override the `assemble` param?

Reply:
It depends:
The test parameter block (`cw310_params(....)`) can override the rom_ext parameter to put a different ROM_EXT into slot A.  The test could also override the `assemble` param to change the ordering or number of images assembled into the final image.

One could imagine something like the following: (assume the test program is either PIC or linked for the virtual window, but the ROM_EXTs are built for side_a/side_b)
```
opentitan_test(
    ...
    cw310 = cw310_params(
        binaries = { "//label/of:rom_ext_b", "rom_ext_b" },
        assemble = "{rom_ext}@0 {firmware}@0x10000 {rom_ext_b}@0x80000 {firmware}@0x90000",
    )
)
```